api/files/views.py

```diff
@@ -35,7 +35,235 @@ def get_file(self, check_permissions=True):
 
 
 class FileDetail(generics.RetrieveUpdateAPIView, FileMixin):
-    """Details about a specific file.
+    """Details about files and folders. *Writeable*.
+
+    Welcome to the Files API.  Brace yourself, things are about to get *weird*.
```

Comment:
This should be less weird within the hour. Once Chris has waterbutler updated, the response from file/folder creation should look a *lot* like one of our json api file responses, except it won't have the info link for files.